src/pages/ticket/tickets.jsx

```diff
@@ -54,7 +78,13 @@ export const TicketList = () => {
                         </InnerContainer>
                         <InnerContainer flex="1" />
                         <InnerContainer flex="2">
-                            Graf over billetter kommer...
+                            Graf over billetter:
+                            <FlexBar>
+                                <BarElement color="blue" title={"Gratisbilletter - " + ticketsFree.length} width={ticketsFree.length} />
```

Comment:
Pretty interesting use of JSX here!

Reply:
Is this something we can use, or should I redo it?

Reply:
I think we use it!